catalog/resourcemanagement/resourcemanagement-usage-ui/src/main/webapp/templates/dataUsageTable.handlebars

```diff
@@ -16,7 +16,7 @@
             <th>User</th>
             <th>Data Used</th>
             <th>Data Remaining</th>
-            <th>Data Limit</th>
+            <th class="data-limit-td">Data Limit</th>
         </tr>
     </thead>
     <tbody class="usertabledata">
```

Comment:
On the `td` elements in this table, it would be good to add a title attribute that is the same as the text within the cell.  That way if the cell overflows, the user can still access the full value.